### DISCUSSION TITLE
Tournament Client only retrieves osu!std SR for converted beatmaps

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16319] [Q&A]
After streaming [this match](https://www.twitch.tv/videos/1250983363?t=00h15m57s) on Twitch, the commentators noticed that the SR being displayed by the lazer client was that of osu!std, not osu!taiko. Upon further investigation, this appears to apply to any ruleset that isn't "osu!"
Version 2021.1225.0
While the logs are long gone, here's the bracket.json used, simplified to the single match for simplicity: [bracket.json.txt](https://github.com/ppy/osu/files/7806332/bracket.json.txt)

**Comment by peppy** [comment]
To confirm, do you mean convert beatmaps? If so, that hasn't been supported because we haven't used these in a tournament yet. Will require some extra thinking as the existing request flow doesn't support specifying the ruleset.
